packages/safe-core-sdk/src/ethereumLibs/Web3Adapter.ts

```diff
@@ -105,7 +107,7 @@ class Web3Adapter implements EthAdapter {
     return getGnosisSafeProxyFactoryContractInstance(safeVersion, proxyFactoryContract)
   }
 
-  getContract(address: string, abi: AbiItem[]): any {
+  getContract(address: string, abi: any): any {
```

Comment:
Please avoid using `any`.

@germartinez, we should add a linting rule to prevent this.

Reply:
Good catch! As discussed we will deal with it in https://github.com/gnosis/safe-core-sdk/issues/138